libs/common/src/sql-db-entities/facility-utilisation-data/facility-utilisation-data.types.ts

```diff
@@ -6,3 +6,10 @@ export type CreateFacilityUtilisationDataWithoutUtilisationParams = {
   reportPeriod: ReportPeriod;
   requestSource: DbRequestSource;
 };
+
+export type UpdateWithFixedFeeAndReportPeriodParams = {
```

Comment:
Nitpick - see prior comment on naming of this on `libs/common/src/sql-db-entities/facility-utilisation-data/facility-utilisation-data.entity.ts`